consoleApp/src/main/java/com/infoshare/service/UserRegistration.java

```diff
@@ -1,34 +1,39 @@
-package com.infoshare;
+package com.infoshare.service;
 
+import com.infoshare.database.FileDb;
 import com.infoshare.domain.Volunteer;
 import com.infoshare.util.Util;
 import com.infoshare.util.ValidatorEnum;
 
+import java.io.IOException;
 import java.util.Scanner;
 
 public class UserRegistration {
     private static final String REGISTRATION_HEADER = "[Witaj w programie do rejestracji]";
     private static final String REGISTRATION_EMAIL = "Podaj e-mail";
     private static final String REGISTRATION_AVAILABILITY = "Czy jesteś dostępny [Y/N]? ";
+    private static final String REGISTRATION_ERROR = "*** Rejestracja nie powiodła się *** Podany adres e-mail już istnieje w bazie danych. ";
 
-    private static Scanner sc = new Scanner(System.in);
-
-    public static void register(){
+    public void register(){
         System.out.println(REGISTRATION_HEADER);
         Volunteer newVolunteer = new Volunteer(
                 Util.readDataFromConsole(Util.REGISTRATION_NAME,ValidatorEnum.ALPHA),
                 Util.readDataFromConsole(Util.REGISTRATION_LOCATION,ValidatorEnum.ALPHA ),
                 Util.readDataFromConsole(REGISTRATION_EMAIL, ValidatorEnum.EMAIL),
                 Util.readDataFromConsole(Util.REGISTRATION_PHONE_NUMBER, ValidatorEnum.PHONENUMBER),
                 Util.createTypeOfHelp(),
-                (Util.readDataFromConsole(REGISTRATION_AVAILABILITY, ValidatorEnum.YESNO).toUpperCase().equals("Y"))?
-                        true:false
-
+                Util.readDataFromConsole(REGISTRATION_AVAILABILITY, ValidatorEnum.YESNO).toUpperCase().equals("Y")
         );
-        SavingUtil.saveToFile("../registeredVolunteer", newVolunteer);
-
+        try{
+            FileDb database = new FileDb();
+            if(database.getVolunteer(newVolunteer.getEmail()) == null ){
```

Comment:
Czy wielkość liter ma znaczenie? Moim zdaniem xyz@test.pl i XYz@test.pl to ten sam mail i nie powinien przejść.